lib/mongo-milestone-db.js

```diff
@@ -0,0 +1,181 @@
+const MilestoneDB = require('sharedb').MilestoneDB;
+const mongodb = require('mongodb');
+
+class MongoMilestoneDB extends MilestoneDB {
+  constructor(mongo, options) {
+    if (typeof mongo === 'object') {
+      options = mongo;
+      mongo = options.mongo;
+    }
+
+    options = options || {};
```

Comment:
NB: The motivation for deleting these options is because the Mongo client validates them. In fact, if you pass in the option `validateOptions`, then your mongo connection will actually fail if we don't tidy these up.

https://github.com/mongodb/node-mongodb-native/blob/bd4fb531a7f599bb6cf50ebbab3b986f191a7ef8/lib/mongo_client.js#L53-L57

I'll add a comment in the code for this, too.